actor/v7action/service_instance_details.go

```diff
@@ -141,6 +148,12 @@ func (actor Actor) getServiceInstanceParameters(serviceInstanceGUID string) (Ser
 	return ServiceInstanceParameters{Value: params}, warnings
 }
 
+type UsageSummaryWithSpaceAndOrg struct {
```

Comment:
Given this type is used first on line 20, it might make sense for it to be defined closer to line 20